### PR TITLE
Added meta tags, title and fevicon

### PR DESCRIPTION
**Pull request by Diyatayal**
Changes:
-Updated the title tag
-Added images for fevicon and meta banner
-Added meta tags, including OG and Twitter tags
-Added fevicn link tag

Test cases checked

Verified the title showing up

### CODE REVIEW

File: reset [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[33mcommit d483c2887610c2fac22284bd2c148beb442f157b[m[33m ([m[1;36mHEAD[m[33m -> [m[1;32mmain[m[33m)[m
+Author: Diyatayal <128578106+Diyatayal@users.noreply.github.com>
+Date:   Sat Apr 13 09:47:10 2024 +0000
+
```
Comment (2411nihalsingh) [inline reset:4]: areee..what is this chakkar? HTML CSS ?

File: src/index.html [hunk 2]
```diff
@@ -1,8 +1,19 @@
 <html lang="en">
     <head>
-        <title>
-            Jovian-Careers
```
Comment (2411nihalsingh) [inline src/index.html:4]: Make the title a little more Catching

File: src/index.html [hunk 3]
```diff
@@ -1,8 +1,19 @@
...
+    <meta name="description" content="Learn about job openings at Jovian and submit your application now">
+    <meta name="keywords" content="data science, software development, jobs">
+    <meta name="author" content="Jovian">
+    <meta name="robots" content="index, follow">
+    <meta property="og:title" content="Jovian Careers">
+    <meta property="og:description" content="Learn about job openings at Jovian and submit your application now">
+    <meta property="og:image" content="/jovian_meta.png">
+    <meta name="twitter:title" content="Jovian Careers">
+    <meta name="twitter:description" content="Learn about job openings at Jovian and submit your application now">
+    <meta name="twitter:image" content="/jovian_meta.png">
```
Comment (2411nihalsingh) [inline src/index.html:14]: Meta tags are pretty nice bro

**Review by 2411nihalsingh — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 3 above)
Comment (Diyatayal) [reply src/index.html:14]: Thanks bro

**Pull request merged**